docs/scenarios/virtual-machines/README.md

```diff
@@ -0,0 +1,157 @@
+## Virtual machine storage migration
+
+Migration toolkit for containers now supports storage migration for KubeVirt virtual machines. Migration can happen offline (virtual machine is turned off) as well as online (virtual machine is running).
+
+Main use cases for this functionality is migrating from one storage class to another. For instance if one is rebalancing across different storage providers, or some new storage is available that is better suited for the workload running inside the VM.
+
+### Supported PV Actions
+
+| Action | Supported | Description |
+|-----------|------------|-------------|
+| Copy | Yes | Create new PV in *same namespace*. Copy data from source PV to target PV, and modify Virtual Machine definition to point to the new PV. If liveMigrate flag is set, the VM will live migrate. If not set, the VM is shutdown, the source PV contents are copied to the target PV and the VM is started again |
```

Comment:
should be PVC instead?

Reply:
No, it should be PVs in this context. The UI refers to these as PVs everywhere so I wanted to be consistent with that in the documentation.